e2e/core/pages/navigation-bar.page.ts

```diff
@@ -172,8 +179,14 @@ export class NavigationBarPage {
     }
 
     async clickLogoutButton(): Promise<void> {
+        Logger.log('Logout');
         await BrowserActions.closeMenuAndDialogs();
         await BrowserActions.clickExecuteScript('.app-sidenav-link[adf-logout]');
+
+        try {
+            await BrowserVisibility.waitUntilElementIsVisible(this.logoutSection);
+        } catch (error) {
```

Comment:
Since we're using the logger already perhaps we could use it to show the error thrown here so it's easier to spot in the future where it's failing?